### ISSUE TITLE
No way to quit without saving?

### ISSUE THREAD

**Issue by RadarNyan** [cheat-engine/cheat-engine#900]
I haven't used Cheat Engine for some years, today I needed it for something, when I'm done I tried to quit but I was given a save file dialog, Cheat Engine won't quit if I cancel or close the dialog, it seems the only way to quit normally would be save a file.

I've searched in the options but failed to find anything related. I understand the design is intended to make user not lose their process, but don't you think it should just be a prompt? The current way is really annoying, surely I can save a file and delete it later, but this forced saving behavior when I explicit not want to (and the table is empty, I even clicked on "new scan" to clear current scan too, basically there's nothing to save) makes me very angry. I ended up pulling out Task Manager and straight off killed it, not because it's easier to do than simply deleting a file, just because I don't want a program force me to do something I don't want.

In almost any other program that has a feature asking user to save their process on exiting, it's always a prompt - just ask the user if they want to save it, if not just throw it away. I've used a number of programs and never faced a program that don't allow me to exit unless I save a file. This kind of behavior is only seen on prank program or virus, why did you design it this way?

**TL;DR** Please make the save prompt behave normally like any other program, or at least add a "Quit without saving" option in the menu.

**Comment by RadarNyan** [comment]
> When you close CE you get a dialog stating :"You haven't saved your last changes yet. Save now?" With 3 buttons :"Yes" , "No", and "Cancel"
> 
> Click on "No" and CE will close

That's what I expect (any program) would do, but it's not what I observed. In my case, I didn't get the dialog you mentioned, instead the "Save file as" dialog was given straight to me and either closing the dialog or press cancel on it would result in canceling the exiting process.

> Did you install some extra extensions to CE that add extra features?
> What was the title of the save dialog window ? And what did you do beforehand ?

What I did was just download CE from the official site (version 7.0), nothing addition was added.
I can still reproduce this bug right now:
![ce_bug gif](https://user-images.githubusercontent.com/2091529/66532872-ea732500-eb4b-11e9-9485-45439ea6f420.gif)

**Comment by cheat-engine** [comment]
Run the reset settings program in CE and see if that helps
If not, install CE to a completely different location (in case there are modified autorun folders in the current installed location)

And besides that, check for any program on your system that might 'help' you by clicking on the default button in dialog windows

**Comment by Coverlin** [comment]
@cheat-engine 
Hi,
Can confirm that CE is working as intended , both 7.0 and earlier

But , as a fellow Sandboxie user as RadarNyan who forgot to mention that this issue happens while inside SB (the blue border on mouseover the title was the indication) , would also like a force close option :)

**Comment by cheat-engine** [comment]
This lua command will work
```lua
getApplication().Terminate()
```

**Comment by mgrinzPlayer** [comment]
so `closeCE()` to close
and `getApplication().terminate()` to force close.
